example-config.yml

```diff
@@ -349,8 +349,6 @@ itinerary:
   disableMetroSeperatorDot: false
   # Shows the duration of a leg below the leg in the metro itinerary summary
   showLegDurations: false
-  # Allows calorie counts to be hidden
-  displayCalories: true
```

Comment:
Was this behavior removed?

Reply:
Displaying calories was completely removed from OTP-UI, so this doesn't apply anymore.